Makefile

```diff
@@ -98,6 +101,7 @@ define build_runtime
 		--build-arg IMAGE_BASE=$(image_base) \
 		--build-arg IMAGE_TAG=$(image_tag) \
 		--build-arg CODE_PATH=$(4) \
+		--build-arg NPM_CONFIG_STRICT_SSL=false \
```

Comment:
It's a build-arg, and we don't define this in the dockerfile.
I think you can turn off the npm SSL check before the `make build`.
refer: https://github.com/api7/apisix-build-tools/blob/master/utils/install-common.sh#L161

Reply:
I create a PR to fix it.
refer: https://github.com/api7/apisix-build-tools/pull/373